lib/color-formatters.spec.js

```diff
@@ -0,0 +1,60 @@
+const assert = require('assert');
+const {
+  coveragePercentage,
+  colorScale,
+  age
+} = require('./color-formatters');
+
+describe('Color formatters', function() {
+  it('should step appropriately', function() {
+    const byPercentage = colorScale([Number.EPSILON, 80, 90, 100]);
```

Comment:
What's the purpose of `Number.EPSILON` here?

Reply:
The semantics of `colorScale` are that the color shifts into the new color when the value >= the provided step value. By making this `EPSILON`, a value of `0.5` produces `yellow`. If it were `1` it would be `red`.

Reply:
Can you just use `0`, or do you want to differentiate 0 from "any possible value after 0"?

Reply:
The existing percentage function makes 0 red and anything after that yellow. I don't see the point in differentiating between 0 and 1, which both seem terrible, though I was aiming for parity.

Reply:
I added tests at `0.5` to clarify this.